spec/models/rental_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Rental, type: :model do
-  let!(:user) { User.create(username: 'robbie', password: 'password') }
+  let!(:user) { User.create(username: 'rossedfort', password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024') }
```

Comment:
Line is too long. [185/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.